lib/permutat.g

```diff
@@ -444,39 +444,26 @@ SetOne( PermutationsFamily, () );
 
 #############################################################################
 ##
-#F  ListPerm( <perm>[, <length>] )  . . . . . . . . . . . . .  list of images
+#F  ListPerm( <perm>[, <n>] )  . . . . . . . . . . . . .  list of images
 ##
 ##  <#GAPDoc Label="ListPerm">
 ##  <ManSection>
-##  <Func Name="ListPerm" Arg='perm[, length]'/>
+##  <Func Name="ListPerm" Arg='perm[, n]'/>
 ##
 ##  <Description>
 ##  is a list <M>l</M> that contains the images of the positive integers
```

Comment:
Although I know this wasn't introduced by your PR @fingolfin, this manual entry seems to start mid-sentence, and could probably be improved a little more.

Reply:
In principle I agree, and I actually even at first did that -- but really all the entries around it are written in this style. Clearly the intention is that the `ListPerm` is read as first word of the text, as it appears when one looks a the help text in the REPL:
```
  ‣ ListPerm( perm[, n] ) ───────────────────────────────────────────────────── function

  is  a  list  l that contains the images of the positive integers from 1 to n under the
```
I don't like this, but it is the same for the majority of other GAPDoc entries in this file. I think it would be worth changing this, but I'd rather do them all at once and in a dedicated PR